src/IdentityUpdates.sol

```diff
@@ -1,14 +1,18 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.28;
 
-import { AccessControlUpgradeable } from
-    "../dependencies/openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
-import { Initializable } from
-    "../dependencies/openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
-import { PausableUpgradeable } from
-    "../dependencies/openzeppelin-contracts-upgradeable/contracts/utils/PausableUpgradeable.sol";
-import { UUPSUpgradeable } from
-    "../dependencies/openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
+import {
+    AccessControlUpgradeable
+} from "../dependencies/openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
+import {
+    Initializable
+} from "../dependencies/openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
+import {
+    PausableUpgradeable
+} from "../dependencies/openzeppelin-contracts-upgradeable/contracts/utils/PausableUpgradeable.sol";
+import {
+    UUPSUpgradeable
+} from "../dependencies/openzeppelin-contracts-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
```

Comment:
We might want to increase formatter line size, because this is really ugly.

Reply:
We can do 140, but I have an idea. Will fix.